PLUB/Configuration/CommonClass/InputTextView.swift

```diff
@@ -0,0 +1,227 @@
+//
+//  InputTextView.swift
+//  PLUB
+//
+//  Created by 김수빈 on 2023/01/08.
+//
+
+import UIKit
+import RxSwift
+
+import SnapKit
```

Comment:
RxSwift도 써드파티라서 다음과 같이 적용하시면 됩니다!
```suggestion
import UIKit

import RxSwift
import SnapKit
```

Reply:
앗 제가 제대로 반영을 안했었네요! 바로 수정해서 올리겠습니다!